app/models/plan_cost_decorator.rb

```diff
@@ -104,7 +104,11 @@ def rate_lookup(the_plan, start_on_date, age, member, benefit_group)
       Caches::PlanDetails.lookup_rate(the_plan.id, start_on_date, age)
     end
     value = if the_plan.health?
-      benefit_group.sic_factor_for(the_plan).to_f * benefit_group.group_size_factor_for(the_plan).to_f
+      if constrain_service_areas?
```

Comment:
My only thought here is that it might be more correct to use a different settings entry for this one.

`use_simple_employer_calculation_model` feels more relevant to me here